src/main/java/com/gojek/beast/sink/bq/BqSink.java

```diff
@@ -61,7 +61,7 @@ public Status push(Records records) {
     private InsertAllResponse insertIntoBQ(Records records) {
         Instant start = Instant.now();
         Builder builder = newBuilder(tableId);
-        records.forEach((Record m) -> builder.addRow(recordInserter.of(m)));
+        records.getRecords().stream().filter(record -> !record.getColumns().isEmpty()).forEach((Record m) -> builder.addRow(recordInserter.of(m)));
```

Comment:
If we are removing the empty columns from pushing into BQ, then why are you adding them to records in `ConsumerRecordConverter` class?
`records.add(new Record(offsetInfo, columns));`. Please check if this can be avoided if the columns are empty

Reply:
How are you planning to send Kafka ack if I don't insert a record in `ConsumerRecordConverter`?